Client/src/main/java/io/github/jwdeveloper/tiktok/http/mappers/LiveDataMapper.java

```diff
@@ -65,8 +65,7 @@ public LiveData.Response map(String json) {
                 default -> LiveData.LiveStatus.HostNotFound;
             };
             response.setLiveStatus(statusValue);
-        } else if (data.has("prompts") && jsonObject.has("status_code") &&
-            data.get("prompts").getAsString().isEmpty() && jsonObject.get("status_code").isJsonPrimitive()) {
+        } else if (data.has("prompts") && jsonObject.has("status_code") && data.get("prompts").getAsString().isEmpty() && jsonObject.get("status_code").isJsonPrimitive()) {
```

Comment:
This part of code will be really hard to debbug. Please extract those conditions to own method